src/pages/who-we-are/components/_hero.tsx

```diff
@@ -88,16 +87,12 @@ const StyledMobileQueryImage = styled(QueryImage)`
 `
 const Hero = () => {
     const data = useStaticQuery(query)
-    const title =
-        getWindowWidth() > 576 ? (
-            localize('Who we are')
-        ) : (
-            <>
-                {localize('Who')}
-                <br></br>
-                {localize('we are')}
-            </>
-        )
+    const title = (
+        <Flex tabletS={{ fd: 'column' }}>
+            <Box mr="3.8rem">{localize('Who')}</Box>
```

Comment:
This 2 needs to be in same div, for localization

Reply:
Apparently it was probably a requirement 😅  should we maintain without breaks? @habib-deriv

Reply:
Reverted due to localization 🙏🏼